app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -238,15 +238,39 @@ class DashboardFragment : BaseFragment<FragmentDashboardPlusBinding, DashboardVM
         }
     }
 
-    private fun updateLastUpdateDateAndTime(){
-        if (viewModel.lastUpdateTime.value != null && viewModel.lastUpdateDate.value != null) {
-            dash_card_no_risky_encounter.card_subtitle = resources.getString(
-                R.string.dashboard_body_no_contact,
-                viewModel.lastUpdateDate.value,
-                viewModel.lastUpdateTime.value
-            )
+    private fun updateLastUpdateDateAndTime() {
```

Comment:
tato metoda sa celkom zozlozitila, mozno by bolo fajn ju rozsekat do mensich convenience metod, nech je viac jasne, co sa tam deje.neviem ci to hra rolu, ale mozno by sa mohli na zaciatku assignut vsetky tri hodnoty, co skrati zapis dost signifikantne.